.travis.settings.xml

```diff
@@ -7,4 +7,14 @@
             <password>${env.SONATYPE_PASSWORD}</password>
         </server>
     </servers>
+<profiles>
```

Comment:
This file is not properly indented

Reply:
Thanks @liubao68 , I have updated the file with proper indentedation